### PR TITLE
Fix/deleted log

### PR DESCRIPTION
**Pull request by andrystylist**
removing unnecessary concole.log

### CODE REVIEW

File: src/components/Modal/ControlledModal.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const stopPropagation = (event) => {
+    event.stopPropagation()
+  }
+
+  const handleClose = () => {
+    onClose()
+  }
+
+  useEffect(() => {
+    const handleScape = (event) => {
```
Comment (ht1204) [inline src/components/Modal/ControlledModal.jsx:21]: handleEscape

File: src/components/data.js [hunk 2]
```diff
@@ -62,8 +62,8 @@ export const dataTimetable = () => {
 
 export const dataCountries = () => {
   const countries = [
-    { id: 3, name: 'Peru' },
-    { id: 4, name: 'Colombia' }
+    { id: 'clmiy14yb0002n238adlw2rkt', name: 'Peru' },
```
Comment (ht1204) [inline src/components/data.js:65]: This should not be here

File: src/pages/Cart/Cart.jsx [hunk 3]
```diff
@@ -55,7 +57,11 @@ function Cart () {
 
   const handleSubmitOrder = (event) => {
     event.preventDefault()
-    // TODO: Navegar programáticamente a Checkout si todo está bien
+    console.log(authData)
```
Comment (ht1204) [inline src/pages/Cart/Cart.jsx:60]: Remove this

**Review by ht1204 — APPROVED** [review]

**Review by jlvillegas04 — APPROVED** [review]
LGTM

**Pull request merged**